.github/workflows/cpp-python-tests.yml

```diff
@@ -7,102 +7,116 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-20.04
     strategy:
       matrix:
         python-version: [3.8]
```

Comment:
Test multiple Python versions ? (3.9,3.10)

Reply:
This is a good idea, but I don't know if the binding will be built with the right python version.

Reply:
https://github.com/introlab/opentera-webrtc/runs/5337177335?check_suite_focus=true#step:14:23
Looks like it works as expected